accl_bias: [-2.541270960612699e-08, -2.72529958581473e-16, -0.009999999999967759]
gyro_bias: [6.054188440025572e-16, -5.108811796714566e-18, 2.334643810688532e-10]